patroni/api.py

```diff
@@ -377,8 +381,10 @@ def get_postgresql_status(self, retry=False):
                                        pg_xlog_location_diff(pg_last_xlog_receive_location(), '0/0')::bigint,
                                        pg_xlog_location_diff(pg_last_xlog_replay_location(), '0/0')::bigint,
                                        to_char(pg_last_xact_replay_timestamp(), 'YYYY-MM-DD HH24:MI:SS.MS TZ'),
-                                       pg_is_in_recovery() AND pg_is_xlog_replay_paused()""", retry=retry)[0]
-            return {
+                                       pg_is_in_recovery() AND pg_is_xlog_replay_paused(),
+                                       (SELECT json_agg(row_to_json(ri)) FROM replication_info ri)""", retry=retry)[0]
```

Comment:
Yeah, 9.2 has only rudimentary json support and this query will fail, because `json_agg` was introduced only in 9.3.
We can replace `json_agg` it with `array_to_json(array_agg(row_to_json(ri)))` and make the query 9.2 compatible, but fairly speaking we had never tested it with version < 9.3, because of pg_rewind.

You are right, it is worth to mention in the README